backend/src/common/plugins/authorization/authorization.plugin.ts

```diff
@@ -51,10 +51,7 @@ const authorizationPlugin: FastifyPluginCallback<AuthOptions> = (
         const [, token] = authorization?.split(' ') ?? [];
 
         if (!token) {
-            throw new HttpError({
-                message: ErrorMessage.UNAUTHORIZED_USER,
-                status: HttpCode.UNAUTHORIZED,
-            });
+            return;
```

Comment:
Are you sure it the correct way to fix it ?
In this case when we don't have token we will not get info about it in frontend when we do request to not white route, did you test this case ?

Reply:
I will check it.

Reply:
You was right. I added checks for auth pages and disabled the error throw only for them.